py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2002,6 +2002,7 @@ def aten_ops_div(
 @dynamo_tensorrt_converter(
     torch.ops.aten.pow.Tensor_Scalar, supports_dynamic_shapes=True
 )
+@dynamo_tensorrt_converter(operator.pow, supports_dynamic_shapes=True)
```

Comment:
From the PR (https://github.com/pytorch/TensorRT/pull/2918), it looks like dynamic testing and registration for `operator.mul` were added because PyTorch internally uses `operator.mul` when using `torch.nn.Linear` layers. This might be why `operator.mul` was registered with the converter. 

If I'm misunderstanding, could you please explain? (cc. @peri044 )

Then, in this PR, why do we need to register `operator.pow`, `operator.eq`, and `operator.mod` with the converter?

Reply:
It seems `operator.*` overrides original Python ops, which results in running any Python op will create a TRT Layer to handle it. However, they are not listed in [the schema](https://github.com/pytorch/pytorch/blob/a408920817ac3852d27fc96ba85109ff4c981d52/aten/src/ATen/native/native_functions.yaml). I'm also curious if these ops' converters are needed

Reply:
Thanks @chohk88 / @zewenli98
If operator.pow is missing and ** is used in module, there is exception from TRTInterpreter.

E torch_tensorrt.dynamo.conversion._TRTInterpreter.UnsupportedOperatorException: Conversion of function _operator.pow not currently supported!

I saw such usage of operator in openchat model. I think we need to register these operators to support such usage.
https://github.com/imoneoi/openchat/blob/master/ochat/models/unpadded_llama.py#L105

Reply:
Thanks for the example. If then, it seems to be necessary I think. 
cc: @narendasan @dheerajperi

Reply:
Thanks for the example. LGTM!